build/update-crdb-certs.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
The usage and context for usage of `apply-certs.sh` is documented in build/README.md.  Let's document this script either in an appropriate place in build/README.md (adding a new section for the new task if needed), or else comprehensively document the purpose and usage of the script in the script itself.  I think the documentation would be more accessible to users in README.md.